api/machine_types.go

```diff
@@ -453,6 +453,11 @@ type MachineConfig struct {
 	FlyProxy    *MachineFlyProxy `json:"fly_proxy,omitempty"`
 	Processes   []MachineProcess `json:"processes,omitempty"`
 
+	// Not used by flyctl yet
+	// Standbys enable a machine to be a standby for another. In the event of a hardware failure,
+	// the standby machine will be started.
+	Standbys []string `json:"standbys,omitempty"`
```

Comment:
propagate the field on deploys by adding it to this list https://github.com/superfly/flyctl/blob/9eb8fa4c8846d333ef674f6e9a4bdeabf4b1a29a/internal/command/deploy/machines_launchinput.go#L51-L57

I am working on doing that by default but for now it has to be added there

Reply:
this is not an issue once https://github.com/superfly/flyctl/pull/2070 is merged